Server/src/models/entities/Savedgraphs.ts

```diff
@@ -75,22 +75,9 @@ export const selectSavedGraphsOfUserQuery = (connection: Connection, user: numbe
         .addSelect('graphs.axisZoomStart', 'axisZoomStart')
         .addSelect('graphs.axisZoomEnd', 'axisZoomEnd')
         .addSelect('graphs.axisUnits', 'axisUnits')
-        .innerJoin(Accounts, 'accounts', 'graphs.accountId = accounts.id')
-        .where('accounts.id = :user', { user: user })
-        .getRawMany();
 
-export const selectOneSavedGraphQuery = (connection: Connection, id: number) =>
-    connection.createQueryBuilder(Savedgraphs, 'graphs')
-        .select('graphs.name', 'name')
-        .addSelect('graphs.id', 'id')
-        .addSelect('graphs.datasetIds', 'datasetIds')
-        .addSelect('graphs.datasetColors', 'datasetColors')
-        .addSelect('graphs.datasetShapes', 'datasetShapes')
-        .addSelect('graphs.datasetHiddenStatus', 'datasetHiddenStatus')
-        .addSelect('graphs.axisVariable', 'axisVariable')
-        .addSelect('graphs.axisLog', 'axisLog')
-        .addSelect('graphs.axisZoomStart', 'axisZoomStart')
-        .addSelect('graphs.axisZoomEnd', 'axisZoomEnd')
-        .addSelect('graphs.axisUnits', 'axisUnits')
-        .where('graphs.id = :id', { id: id })
-        .getRawOne();
+export const selectGraphStateAccountQuery = (connection: Connection, graphId: number) =>

```

Comment:
The names between these 2 are a little confusing. Maybe rename one to be 
selectAllGraphStates
selectGraphStateByAccountId

Reply:
I will rename the second to selectGraphOwnerQuery which should be more clear.